tracee-ebpf/tracee/events_processor.go

```diff
@@ -142,9 +146,13 @@ func (t *Tracee) processEvent(ctx *context, args map[string]interface{}, argMeta
 				}
 				castedSourceFileCtime := int64(sourceFileCtime)
 
-				capturedFileID := fmt.Sprintf("%d:%s", ctx.MntID, sourceFilePath)
+				containerId := t.containers.GetCgroupInfo(ctx.CgroupID).ContainerId
```

Comment:
You already have containderid initialized above in this function. Why doing it again?

Reply:
We do not need it for every case + every time we call the function only one case is matched. I'm okay with taking it apart at the beginning of the switch statement but I don't think is necessary.